scripts/checker.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+set -u
+set -o pipefail
+
+
+check_exit_code() {
+    exit_code=$?
+    # full path to output file
+    #local checker_file="/var/lib/container_probes/checker_ready.txt"
+
+    # analyze exit code with trapped function
+    if [ ${exit_code} -ne 0 ]; then
+        # Clear the readiness file
+        echo "ERROR: Liveness probe failed with code ${exit_code}" # | tee "${checker_file}"
+        exit_code=1
+    else
+        # TODO: implement robust readiness check
+        #./checker_ready.sh
+        echo "READY" # | tee ${checker_file}
+    fi
+    exit ${exit_code}
+}
+
+# Capture Exit Code with trapped pseudo signal EXIT
+# When the script is completed or exits for any reason, the
+# commands in the check_exit_code function will be executed.
+trap check_exit_code EXIT
```

Comment:
I would remove this code for the moment, this will/might be needed when we have the readiness probe. I would keep the minimum for liveness probe for the moment, we can add this later when doing the readiness probe.

Reply:
if searchd is not running the container should be restarted by docker or autoheal container. thats why i would like to call the function searchd with the the trapped function here